manager/engineimage.go

```diff
@@ -117,6 +117,10 @@ func (m *VolumeManager) CheckEngineImageReadiness(image string, nodes ...string)
 	return m.ds.CheckEngineImageReadiness(image, nodes...)
 }
 
-func (m *VolumeManager) CheckEngineImageReadinessForVolume(image, volumeName, nodeID string) (isReady bool, err error) {
```

Comment:
Do we need these? I think you can directly call `m.ds.CheckEngineImageReadyOnAllVolumeReplicas`

Reply:
You are right.  We don't need them. Fixed. Thank you